src/solc_0.8/common/BaseWithStorage/WithMetaTransaction.sol

```diff
@@ -8,7 +8,7 @@ contract WithMetaTransaction is WithAdmin {
     uint8 internal constant METATX_SANDBOX = 1;
     uint8 internal constant METATX_2771 = 2;
 
-    mapping(address => uint8) internal _metaTransactionContracts;
+    mapping(address => uint8) internal _metaTransactionProcessors;
```

Comment:
Need to think about this. I first considered a name-change here because `ERC1155ERC721.SOL` also has a`mapping(address => uint8) internal _metaTransactionContracts;` which can't be removed due to upgradability.
We need to decide if `ERC1155ERC721.SOL` should inherit `WithMetaTransaction.sol` for consistency, or just include the bare minimum metaTx functionality needed to keep bytecode size down.